person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.ran(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+  private :of_age?
```

Comment:
- [ ] This private method declaration does not work because it is done before the `of_age?` method is defined. For this to work kindly move this line to the bottom of the class.